### DISCUSSION TITLE
Version libre isn't update on f-droid

### DISCUSSION THREAD

**Discussion by marcossmtp** [deckerst/aves#723] [General]
I don't know why the version aves libre is not update in f-droid while the other version is.
Someone knows the reason?

**Comment by deckerst** [comment]
APK too large?

**Reply by marcossmtp** [reply]
wow, that's a reason not to update an app in f-droid?
I didn't know that reason exist, then why the normal version that increase the load hasnt that problem.
If I update with the github apk the sign will be the same of f-droid to continue after that updating from f-droid?
I tried to update with the apk of github and give an error with the package.
thanks

**Reply by deckerst** [reply]
I don't know the reason, it was just an idea. I know Izzy limits apk size, but i don't know F-droid policy. Also, it often takes several days for F-droid to roll out new releases, so maybe you just need to wait a little bit.

And about updating the app already on your device, i suppose it doesn't work because the releases on GitHub and the ones on F-droid are not signed by the same entity.

**Reply by marcossmtp** [reply]
Ok , thanks very much
